src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+  @Override
+  public void doWork() {
+    System.out.println("Excavator started its work");
```

Comment:
The message should indicate what type of `Machine` is working. Consider specifying that an `Excavator` is working to make the message more informative according to the checklist.